examples/src/main/scala/org/apache/spark/examples/ml/MultilayerPerceptronClassifierExample.scala

```diff
@@ -32,6 +32,7 @@ object MultilayerPerceptronClassifierExample {
   def main(args: Array[String]): Unit = {
     val spark = SparkSession
       .builder
+      .master("local[4]")
```

Comment:
Oh sorry `master()` should not be added for examples, this was for my own testing and I mistakenly committed it...reverting this change